tests/conftest.py

```diff
@@ -64,3 +66,71 @@ def ensure_default_subclasses(db):
         obj2.set_current_language("lt")
         obj2.title = "Duomenų publikavimo paslauga"
         obj2.save()
+
+
+# TODO: remove this after the pipeline starts running migrations before testing
+@pytest.fixture(autouse=True)
```

Comment:
```suggestion
@pytest.fixture(scope="session", autouse=True)
```

It will execute this fixture once per test run instead of running before each test case (function). Can increase performance drastically. Once caveat - it will work well only if these records are not modified during test run (used as read-only)

Reply:
Using scope="session" blocks the fixtures ability to access the database, so added a `django_db_blocker.unblock()`